File: src/test/java/pl/patrykdolata/lifemanager/LifeManagerApplicationTests.java
package pl.patrykdolata.lifemanager;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.core.env.Environment;

import java.util.Arrays;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.hasItemInArray;

@SpringBootTest
class LifeManagerApplicationTests {

    @Autowired
    private Environment env;

    @Test
    void contextLoads() {
        assertThat(env.getActiveProfiles(), hasItemInArray("test"));
    }

}

File: src/test/kotlin/pl/patrykdolata/lifemanager/LifeManagerApplicationTests.kt
package pl.patrykdolata.lifemanager

import org.hamcrest.MatcherAssert.assertThat
import org.hamcrest.Matchers.hasItemInArray
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.core.env.Environment

@SpringBootTest
class LifeManagerApplicationTests {

	@Autowired
	private lateinit var env: Environment

	@Test
	fun contextLoads() = assertThat(env.activeProfiles, hasItemInArray("test"))

}
